doc/devel/gitwash/known_projects.inc

```diff
@@ -1,12 +1,12 @@
 .. Known projects
 
 .. PROJECTNAME placeholders
-.. _PROJECTNAME: http://nipy.org
+.. _PROJECTNAME: https://nipy.org/
```

Comment:
We don't maintain gitwash so all these changes will get overwritten...

Reply:
Please revert all changes to gitwash.